client/src/components/InputRating/InputRating.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { ChangeEvent, FC, InputHTMLAttributes } from 'react';
+import { BsStar, BsStarFill } from 'react-icons/bs';
+import styles from './InputRating.module.scss';
+
+type InputRatingProps = {
+  onChange?: (event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+  value: number;
+};
+
+const InputRating: FC<InputHTMLAttributes<HTMLInputElement> & InputRatingProps> = ({
+  type,
+  name,
+  id,
+  value,
+  onChange
+}) => {
+  return (
+    <div className={type !== 'hidden' ? styles.ratingContainer : styles.ratingContainerMsg}>
+      <div className={styles.textContainer}>
+        <h3>{type !== 'hidden' ? '* Rating' : ''}</h3>
+        <div>{type !== 'hidden' && 'Fields marked with a * are required.'}</div>
+      </div>
```

Comment:
do you need to render h3, div tags and `textContainer` if type is hidden? if no, please update the code

Reply:
Looks like you reverted changes i talked about on previous review :)

Reply:
Also you could use function you already created instead of typing manually “* Rating” but it can be fixed in next PR

Reply:
Also you could use function you already created instead of typing manually “* Rating” but it can be fixed in next PR